### DISCUSSION TITLE
Third-party package updates

### DISCUSSION THREAD

**Discussion by congcijueqi** [e3nn/e3nn#364] [Q&A]
For the change_of_basis_matrix function in lie_learn.representations.SO3.irrep_bases.c file in which new package can you find

**Comment by mariogeiger** [comment]
I don't know. In e3nn we only use one basis (which is real).

In e3nn there is this function https://github.com/e3nn/e3nn/blob/19d0377da7925b684bade615ecefbc12627076a2/e3nn/o3/_wigner.py#L28 that gives the change of basis from one of the complex basis to the real basis we use in e3nn.
